packages/app/src/vis-packs/core/complex/MappedComplexLineVis.tsx

```diff
@@ -62,19 +63,26 @@ function MappedComplexLineVis(props: Props) {
     showGrid,
   } = config;
 
+  const { phaseArrays, amplitudeArrays } = usePhaseAmplitudeArrays([
+    value,
+    ...auxValues,
+  ]);
   const numAxisArrays = useToNumArrays(axisValues);
-  const [dataArray, ...auxArrays] = useMappedComplexArrays(
```

Comment:
This was a very complicated hook that tried to do way too many things. It really didn't help me keep my sanity when I was looking for a solution to the mixed `NXdata` problem... Looking at it now, I wish I had done this refactoring first...